src/Equinor.ProCoSys.IPO.Command/InvitationCommands/UpdateAttendedStatusOnParticipant/UpdateAttendedStatusOnParticipantCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿using MediatR;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Command.InvitationCommands.UpdateAttendedStatusOnParticipant
+{
+    public class UpdateAttendedStatusOnParticipantCommand : IRequest<Result<string>>, IInvitationCommandRequest
```

Comment:
Add unittest to AccessValidatorTests
(Should be done for all new commands or requests inheriting either IInvitationCommandRequest or IInvitationQueryRequest)